pkg/api/resources/Cluster.go

```diff
@@ -204,7 +205,7 @@ func DeleteCluster(name string, clientset simple.Clientset) error {
 		clusterResources[k] = resource
 	}
 	if len(clusterResources) != 0 {
-		err = ops.DeleteResources(cloud, clusterResources)
+		err = ops.DeleteResources(cloud, clusterResources, 0, 10*time.Second, 10*time.Minute)
```

Comment:
Not sure about those values. I think this is the defaults. Might make sense to add those as parameters somehow.